lib/kernel-manager.coffee

```diff
@@ -176,3 +177,6 @@ module.exports = KernelManager =
 
     destroy: ->
         _.forEach @runningKernels, (kernel) -> kernel.destroy()
+
+    getStartupCode: (language) ->
+        return @getConfigJson('startupCode')[language] + ' \n'
```

Comment:
Thanks for noticing! I added the line break at some point and didn't thought about this.